integrated_channels/integrated_channel/tasks.py

```diff
@@ -119,13 +119,19 @@ def remove_duplicate_transmission_audits():
         duplicates_found += num_duplicates
         # Mysql doesn't support taking the count of a sliced queryset
         duplicates_to_delete = duplicates[1:]
-        if getattr(settings, "DISABLE_REMOVE_DUP_TRANSMISSION_AUDIT_DRY_RUN", False):
-            for duplicate in duplicates_to_delete:
-                duplicate.delete()
-        else:
+        LOGGER.info(
+            f"remove_duplicate_transmission_audits task dry run mode" +
+            f"set to: {getattr(settings, 'DRY_RUN_MODE_REMOVE_DUP_TRANSMISSION_AUDIT')}"
```

Comment:
imho i'd resolve the getattr into a variable once and use that variable in the log message and in the if statement...

```
dry_run_flag = getattr(settings, "DRY_RUN_MODE_REMOVE_DUP_TRANSMISSION_AUDIT", True)
logger.info(....dry_run_flag....)
if dry_run_flag:
```